src/components/input-panel/compiled-spec-header/index.tsx

```diff
@@ -73,6 +127,34 @@ class CompiledSpecDisplayHeader extends React.PureComponent<Props> {
             </>
           )}
 
+          {this.props.sidePaneItem === SIDEPANE.Config && (
+            <>
+              <ArrowUpCircle
```

Comment:
Please move the two buttons together so they don't get stretched apart when I resize the panel on the left.